app/(main)/(routes)/documents/[documentId]/page.tsx

```
@@ -1,45 +1,58 @@
-            <div>
-                Loading...
-            </div>
-        )
-    }
-    if(document===null){
-        return <div>    Not Found  </div>
-    }
+  const update = useMutation(api.document.update);
 
+  const onChange = (content: string) => {
+    // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)